backend/alembic/versions/e1e7929bebd8_initial_migration.py

```
@@ -0,0 +1,60 @@
+# revision identifiers, used by Alembic.
+revision: str = 'e1e7929bebd8'
+down_revision: Union[str, Sequence[str], None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('quiz',
+    sa.Column('id', sa.Integer(), nullable=False),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)